impl/maven-cli/src/main/java/org/apache/maven/cling/invoker/LookupInvoker.java

```diff
@@ -305,16 +307,29 @@ protected BuildEventListener doDetermineBuildEventListener(C context) {
     }
 
     protected void createTerminal(C context) {
+        boolean stdStreamsRedirected = System.console() == null
```

Comment:
I don't think those boolean are needed.
`System.console()` kinda delegates to JDK's internal shaded JLine.